android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java

```diff
@@ -135,10 +137,16 @@ public void onLayoutChange(View v, int left, int top, int right, int bottom, int
 		} else {
 			tv.setGravity(Gravity.TOP | Gravity.LEFT);
 		}
+		tv.setBackgroundColor(Color.TRANSPARENT);
```

Comment:
Question:
Will this remove the underline even if no custom background was applied?

Reply:
I just confirmed that the underline is no longer shown. We should only hide it if a custom background color/image is applied.